packages/onboarding/src/mshots-image/index.tsx

```diff
@@ -171,13 +181,18 @@ const MShotsImage = ( {
 		visible ? 'mshots-image-visible' : 'mshots-image__loader'
 	);
 
-	// The "! visible" here is only to dodge a particularly specific css
```

Comment:
Note: I removed the logic described in this comment because the CSS rule that was mentioned no longer appears to exist, and didn't come across any visual glitches.